README.md

```diff
@@ -1,4 +1,5 @@
-# Roll For Reaction
+<img src="https://user-images.githubusercontent.com/45865186/77014224-a9265180-69d6-11ea-92d3-18af1d76c740.png" width="400">
+
```

Comment:
Why are we not using relative links? This is the recommended way to do images in markdown.